tests/functional_tests/fitting/test_joint-fit.py

```diff
@@ -90,7 +92,7 @@ def test_joint_fit_neutron_xray_pd_cwl_pbso4() -> None:
     project.analysis.show_meas_vs_calc_chart(expt_id="xrd", x_min=26, x_max=28, show_residual=True)
 
     # Compare fit quality
-    assert_almost_equal(project.analysis.fit_results.reduced_chi_square, 21.1, decimal=1)
+    assert_almost_equal(project.analysis.fit_results.reduced_chi_square, 14.4, decimal=1)
```

Comment:
Note that I had to adjust the expected chi_squared after changing the target sum of weights.